src/webview/css/css.js

```diff
@@ -260,12 +269,12 @@ hr {
   object-fit: contain;
 }
 blockquote {
-  padding-left: 0.5em;
+  -webkit-padding-start: 0.5em;
```

Comment:
I did research this a bit and it is an interesting CSS property. Would be nice to mention somewhere in the commit message 
about things like (that I now know of but other people will not):
 * why do we use this
 * does Safari and Chrome both support this one
 * is it supported on all devices we do support (my concern will be Android 4.4)